File: OOP/method.cpp
#include <bits/stdc++.h>

using namespace std;

class A
{
private:
   int value;

public:
   void set_value(int v)
   {
      value = v;
   }

   void show_value()
   {
      cout << "The value is: " << value << endl;
   }
};

int main()
{
   A obj;
   obj.set_value(10);
   obj.show_value();
   return 0;
}

File: OOP/method.py
class A:
    def __init__(self):
        self.value = None

    def set_value(self, v):
        self.value = v

    def show_value(self):
        print("The value is:", self.value)


obj = A()
obj.set_value(10)
obj.show_value()
